src/pages/ProposalSubmission/Custom.tsx

```diff
@@ -310,15 +302,21 @@ const NewProposalPage = observer(() => {
             return daoService.encodeControllerGenericCall(
               call.to,
               callData,
-              call.value
+              call.callType === 'simple'

```

Comment:
I would encapsulate this as a function, it's repeated at line 317 and it would help to avoid nested ternary operators

Reply:
Absolutely, not my code though, just a revert :P